setup.cfg

```diff
@@ -0,0 +1,2 @@
+[pycodestyle]
+max-line-length = 160
```

Comment:
is it too difficult to solve this? :thinking: or is it for consistence with rubocop?